packages/eslint-plugin/tests/rules/strict-enums/strict-enums-assignment.test.ts

```diff
@@ -0,0 +1,235 @@
+import rule from '../../../src/rules/strict-enums';
+import {
+  fruit2EnumDefinition,
+  fruitEnumDefinition,
+  strictEnumsRuleTester,
+} from './shared';
+
+strictEnumsRuleTester.run('strict-enums-assignment', rule, {
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/pull/6091#issuecomment-1327759179:

> RE: Refactoring the tests
> The problem with removing the names is that you lose the ability to run specific tests, which I found to be essential when debugging specific parts of the rule + printing out variables.

You can use `only: true`, no? https://eslint.org/docs/latest/developer-guide/unit-tests#running-individual-tests

We generally use that as it gets to be a real chore to make unique names per test.

Reply:
Oh, I see.